scripts/docker/debian/sid/doc.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:stretch
+FROM debian:sid
```

Comment:
Would it work on Debian buster (stable) as well?

Reply:
I tried, but unfortunately Debian buster ships with the same (horribly outdated) version of Doxygen (version `1.8.13`). The [bug in the generator for the PDF manual](https://github.com/doxygen/doxygen/issues/6456) should be fixed with Doxygen `1.8.15` (released about two years ago).

Reply:
Good to know. If we encounter further problems, we can move the doc task to Fedora. I think they use much more recent packages in general.